dll/win32/msi/dialog.c

```diff
@@ -3159,8 +3159,22 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
          * if a width is invalid, all remaining columns are hidden
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
+#ifdef __REACTOS__
+            if ((count == 0) && (!wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 )))
+            {
+                msi_free( num );
+                FIXME("Style prefix not supported. Skipping.\n");
+                continue;
+            }
+            else
```

Comment:
Why this redundant `else` block?

Reply:
In what sense is this "else" block redundant?

Reply:
I think he means the diff can be reduced by removing else case from the condition, but also from the diff guard.

Reply:
Yes, please do so.